Libraries/Azure.psm1

```diff
@@ -213,57 +205,179 @@ Function Validate-SubscriptionUsage($RGXMLData, $Location, $OverrideVMSize, $Sto
     }
 }
 
+Function Get-StorageAccountFromRegion($Region, $StorageAccount)
+{
+	#region Select Storage Account Type
```

Comment:
It looks 8-character not 4

Reply:
it's tab, so it looks like 8-character. After aligned to tab for the whole file, that weird part will disappear.  Who knows what a strange coding rule using tab ! which conflicts with hundreds of language recommended.